components/new_channel_flow/new_channel_flow.tsx

```diff
@@ -137,7 +172,7 @@ export default class NewChannelFlow extends React.Component {
         });
     };
 
-    onCreateChannelError = (err) => {
+    onCreateChannelError = (err: {id: string; message: string}) => {
```

Comment:
I think we have an error type in mattermost-redux...

Reply:
Done :+1: